unreleased/sentry_exporter_statuses.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sentryexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Make sentry status more detailed - use HTTP and Grpc codes from tags
+
+# One or more tracking issues related to the change
+issues: []
```

Comment:
Changelog CI check is failing. You need to add here either an issue number or this PR's number (#13407)

Reply:
Done